.github/subscribe_to_label.json

```diff
@@ -0,0 +1,8 @@
+{
+  "WordPress/gutenberg-design": [
+    "\ud83d\uddbc\ufe0f aspect: design"
```

Comment:
Are these the emojis? They don't render like that in macOS, I wonder if they will match the existing labels 😅

<img width="464" alt="CleanShot 2022-10-11 at 11 21 48@2x" src="https://user-images.githubusercontent.com/9145885/195132813-decb71b9-be0f-4230-a3d3-4de86762139a.png">

Reply:
I pasted the emojis in there, and one of our JSON linting actions changed it to the unicode values 🤷🏼‍♀️